### DISCUSSION TITLE
Is it possible to see a more readable stack trace in the error in the chrome console?

### DISCUSSION THREAD

**Discussion by claytonweaver** [leptos-rs/leptos#2067] [Q&A] [answered]
Noob Question: As I have been following along with the parent [child example](https://leptos-rs.github.io/leptos/view/08_parent_child.html) in the book, I tried seeing what would happen if I try to get a write signal that doesn't exist. I couldn't even see the message that was passed into .expect()

![image](https://github.com/leptos-rs/leptos/assets/58234904/967e8282-667a-4e16-a80b-9b2893a40eec)

Is there a way to enable some kind of source mapping so that we can get the actual line numbers in the stack trace of the panic? Or is that a limitation of WASM?

**Comment by gbj** [comment, marked as answer]
Great question! We should make sure to add it to the docs if it's not there already -- you should set up and use [`console_error_panic_hook`](https://github.com/rustwasm/console_error_panic_hook), which is in all the examples. Super easy, just add the crate as a dependency and `console_error_panic_hook::set_once()` in your main function.

**Reply by claytonweaver** [reply]
Thanks!!
